### ISSUE TITLE
Feature: When comments are collapsed show comment Icon

### ISSUE THREAD

**Issue by SridharPasham** [mrlacey/CollapseComments#15]
### Installed product versions
- Visual Studio: 2019, 16.7
- This extension: 1.9

### Description
When comments are collapsed show comment Icon like in [this extension](https://marketplace.visualstudio.com/items?itemName=3dGrabber.NoComment). 

### Expected behavior
When comments are collapsed show comment Icon.

**Comment by mrlacey** [comment]
Is it not possible to use both extensions?

**Comment by SridharPasham** [comment]
I have observed VS slowness when installed that extension. Also found same comments about that extension. Looks like it is not actively maintained.

**Comment by mrlacey** [comment]
What's the goal here?

Is it that even when collapsed the comments can be distracting and so you'd like them hidden? 
Or is it something else?

**Comment by SridharPasham** [comment]
> Is it that even when collapsed the comments can be distracting and so you'd like them hidden?

Yes. With some sort of indicator indicating there are hidden comments.

**Comment by yannduran** [comment]
Sounds like a great option to have

**Comment by mrlacey** [comment]
What may look like a very simple feature has lots of nuance and hidden complexity.

- How to handle comments of single vs multi-line length?
- What other functionality from that other extension should also be considered?
- How to avoid the performance issues experienced with the other extension? (It needs to do a lot of work to replace the existing/built-in comment experience and provide a custom comment editing experience.) 

I don't expect to get to this any time soon but will consider PRs.
Although, if you end up recreating everything in the NoComment extension it might be easier to have that as a separate extension you publish yourself.

**Comment by ramoneeza** [comment]
May be instead to show an icon, allow to change also the style of collapsed comment. I hate the box around a colapsed comment. And font became the default one.

**Comment by fiseni** [comment]
Hi,

The extension OP mentioned is obsolete, and it's not available for VS2022.
Not necessarily we should copy their approach by displaying a specific icon. Instead, a reasonable compromise would be to print a dummy short "..." text. A new package option may be introduced, let's say `ShortCollapseForm`. Anyone who wants this feature can explicitly opt-in by enabling this setting.

I just found an old PR I created for the Mad's Tweakster extension. You can see the related issue and PR [here](https://github.com/madskristensen/Tweakster/issues/2). Sounds good enough?

**Comment by mrlacey** [comment]
The issue preventing this is that I am unable to find a way to modify what is displayed in the collapsed form. This is provided by other tooling and there doesn't appear to be an API that supports modifying this.

The other extension mentioned in the OP seems to work by identifying comments and then taking responsibility for their display , and that allows it to control whether a large or small visual is displayed. This could be recreated but I don't know how well this would work with the latest version of VS. The original extension targeted a version of VS with different APIs and I don't know how it now handles multiple extensions that are trying to manipulate the same comments.

@fiseni your PR works based on changing the output of the 'Tagger`  in this instance the Tagger is in someone else's (roslyn's?) code and so not something I can directly reuse here.

**Comment by fiseni** [comment]
Yeah, I understand what you're saying. I also gave it a try in this new [PR](https://github.com/madskristensen/Tweakster/pull/127). The built-in tagger somehow takes precedence (or ignores it). I couldn't find any solution either.
